### ISSUE TITLE
Volume permissions errors render as 500s via API

### ISSUE THREAD

**Issue by bfosberry** [getcarina/feedback#55]
These should be 403s:

```
Timestamp: 2016-03-18 18:30:48.680417158 +0000 UTC
Code: System error

Message: permission denied

Frames:
---
0: setupRootfs
Package: github.com/opencontainers/runc/libcontainer
File: rootfs_linux.go@40
---
1: Init
Package: github.com/opencontainers/runc/libcontainer.(*linuxStandardInit)
File: standard_init_linux.go@57
---
2: StartInitialization
Package: github.com/opencontainers/runc/libcontainer.(*LinuxFactory)
File: factory_linux.go@240
---
3: initializer
Package: github.com/docker/docker/daemon/execdriver/native
File: init.go@35
---
4: Init
Package: github.com/docker/docker/pkg/reexec
File: reexec.go@26
---
5: main
Package: main
File: docker.go@18
---
6: main
Package: runtime
File: proc.go@111
---
7: goexit
Package: runtime
File: asm_amd64.s@1721
```

**Comment by bfosberry** [comment]
This happens on container start

**Comment by lblackstone** [comment]
@bfosberry Could you provide some more detail on the command you were trying to run? May be a Docker issue.

**Comment by bfosberry** [comment]
It's not related to the container command, it's related to volume restrictions:

```
bfosberry@slothula:~$ docker run -it -v /tmp:/tmp ubuntu
Timestamp: 2016-03-22 04:01:26.98323625 +0000 UTC
Code: System error

Message: permission denied

Frames:
---
0: setupRootfs
Package: github.com/opencontainers/runc/libcontainer
File: rootfs_linux.go@40
---
1: Init
Package: github.com/opencontainers/runc/libcontainer.(*linuxStandardInit)
File: standard_init_linux.go@57
---
2: StartInitialization
Package: github.com/opencontainers/runc/libcontainer.(*LinuxFactory)
File: factory_linux.go@240
---
3: initializer
Package: github.com/docker/docker/daemon/execdriver/native
File: init.go@35
---
4: Init
Package: github.com/docker/docker/pkg/reexec
File: reexec.go@26
---
5: main
Package: main
File: docker.go@18
---
6: main
Package: runtime
File: proc.go@111
---
7: goexit
Package: runtime
File: asm_amd64.s@1721
docker: Error response from daemon: 500 Internal Server Error: Container command could not be invoked..
bfosberry@slothula:~$ 
```

**Comment by lblackstone** [comment]
Yeah, that's what I was guessing. The Docker daemon is what's actually returning the 500, but it's the security restrictions on the host that are causing the failure. We're got some things in the works now that should improve error messages in the future, especially related to permissions issues. Thanks for the input!

**Comment by bfosberry** [comment]
Perfect Ty! Feel free to close this out if you want to

**Comment by lblackstone** [comment]
@bfosberry A partial fix for this issue was deployed this morning. Docker will still return a 500 status code (fixed in 1.11.2 to be deployed at a future date), but you should get a more helpful error message rather than a stack trace at this point.
